src/sentry/filters/legacy_browsers.py

```diff
@@ -11,14 +14,53 @@
     'Safari': 6,
     'Edge': 0,
     'Opera': 15,
+    'Android': 4,
 }
 
 
+class LegacyBrowserFilterSerializer(serializers.Serializer):
+    # TODO: maybe find a serializer that handles multple values already
+    active = serializers.BooleanField()
+    subfilters = MultipleChoiceField(choices=[
+        'ie8',
+        'ie9',
+        'opera',
```

Comment:
We should probably version these too, e.g. `safari7`, rather than "all of Safari".